lib/components/narrative/default/itinerary-summary.js

```diff
@@ -1,7 +1,9 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 
-import ModeIcon from '../../icons/mode-icon'
+import TriMetLegIcon from '@opentripplanner/icons/lib/trimet-leg-icon'
```

Comment:
Should be at the very least grouped with the 3rd party imports and ideally sorted before React and PropTypes. Also PropTypes should be before React.